manifests/g/Google/AndroidStudio/Beta/2021.3.1.13/Google.AndroidStudio.Beta.installer.yaml

```
@@ -0,0 +1,21 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=QUSU.5-1-22000-653
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Google.AndroidStudio.Beta
+PackageVersion: 2021.3.1.13
```

```suggestion
PackageVersion: 2021.3
```
This must match Control Panel.